okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpChannelBuilder.java

```diff
@@ -122,10 +123,23 @@ public OkHttpChannelBuilder overrideHostForAuthority(String host) {
   }
 
   /**
-   * Provides a SSLSocketFactory to establish a secure connection.
+   * Sets the negotiation type for the HTTP/2 connection.
+   *
+   * <p>Default: <code>TLS</code>
+   */
+  public OkHttpChannelBuilder negotiationType(NegotiationType type) {
+    negotiationType = type;
+    return this;
+  }
+
+  /**
+   * Provides a SSLSocketFactory to replace the default SSLSocketFactory used for TLS.
+   *
+   * <p>By default TLS is enabled, and <code>SSLSocketFactory.getDefault()</code> will be used.
```

Comment:
Don't specify "By default TLS is enabled" here, as it doesn't really relate to this method. Instead, maybe something more like:

> By default, when TLS is enabled <code>SSLSocketFactory.getDefault()</code> will be used.